core/src/main/java/org/apache/iceberg/puffin/FileMetadataParser.java

```diff
@@ -41,6 +41,8 @@ private FileMetadataParser() {
 
   private static final String TYPE = "type";
   private static final String FIELDS = "fields";
+  private static final String SOURCE_SNAPSHOT_ID = "source-snapshot-id";
+  private static final String SOURCE_SNAPSHOT_SEQUENCE_NUMBER = "source-snapshot-sequence-number";
```

Comment:
What is the value of `source` here? What about just using `snapshot-id`?

Also, it should just be `sequence-number` and not `snapshot-sequence-number`.

Reply:
Updated